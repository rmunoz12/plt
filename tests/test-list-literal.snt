@setup { int x = 1; int y = 2; int z = 3; }

@turns {

func void begin() {
    int x = 1; int y = 2; int z = 3;
    str s = "hello world!";

    print([]); print("\n");
    print([x, y, z]); print("\n");
    print([3, 4, 5]); print("\n");
    print(["abc", "def"]); print("\n");
    print([s, "123"]); print("\n");
    print([True, False, True]); print("\n");

    end;
}

}
