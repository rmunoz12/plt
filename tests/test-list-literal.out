[]
[1, 2, 3]
[3, 4, 5]
[abc, def]
[hello world!, 123]
[true, false, true]
